.github/workflows/calibreapp-image-actions.yml

```
@@ -0,0 +1,20 @@
+name: Compress images
+on: pull_request
+jobs:
+  build:
+    name: calibreapp/image-actions
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Compress Images
+        uses: calibreapp/image-actions@master
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          jpegQuality: "80"
+          jpegProgressive: false
+          pngQuality: "80"
+          webpQuality: "80"
+          ignorePaths: "node_modules/**,build"
```

I don't think we need this we don't have node_modules nor build directories. Would this include only `public`?

```suggestion
          ignorePaths: "!public"
```

---

As we don't need to include the images within the **static** folder, I think I will change the ignorePaths to static